app/build.gradle

```diff
@@ -2,8 +2,6 @@ apply plugin: 'com.android.application'
 
 apply plugin: 'kotlin-android'
 
-apply plugin: 'kotlin-android-extensions'
```

Comment:
This will also remove `@Parcelize` support, you can use 

```
androidExtensions {
    features = ["parcelize"]
}
```

If you want to remove synthetics, but keep `@Parcelize`